pkg/disruptors/pod_test.go

```diff
@@ -0,0 +1,157 @@
+package disruptors
+
+import (
+	"context"
+	"sort"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/grafana/xk6-disruptor/pkg/kubernetes"
+	"github.com/grafana/xk6-disruptor/pkg/testutils/kubernetes/builders"
+
+	corev1 "k8s.io/api/core/v1"
+	k8sruntime "k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes/fake"
+)
+
+func Test_NewPodDisruptor(t *testing.T) {
+	t.Parallel()
+
+	testCases := []struct {
+		title       string
+		name        string
+		namespace   string
```

Comment:
What are `name` and `namespace` used for? They don't seem to be used in the test body.

Reply:
There are leftovers from previous versions of the test. Thanks for the catch.